plugins/push/frontend/public/templates/message-editor-with-emoji-picker.html

```diff
@@ -0,0 +1,40 @@
+<div class="cly-vue-push-notification-emoji-wrapper">
+    <validation-provider ref="defaultLocalizationValidationProvider" v-slot="validation" rules="required"> </validation-provider>
+    <div :class="['el-input',hasValidationErrors?'is-error':null]">
+        <component is="div" class="el-input__inner" v-bind:id="id" contenteditable="true" ref="element" @input="onInput($event.target.innerHTML)"></component>
+    </div>
+    <emoji-picker @emoji="appendEmoji" :search="search">
+        <div
+        class="emoji-invoker"
+        slot="emoji-invoker"
+        slot-scope="{ events: { click: clickEvent } }"
```

Comment:
Not a must, but you might check out v-slot[scope-name]="slot-scope" notation :slightly_smiling_face:

Reply:
Good catch. I used most of what I got from the lib, I will switch to slot-scope notation.